#7493 - excel. commands reviewed (#56)
G1ANT-Robot/G1ANT.Addon: G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/ExcelRunVBCodeCommand.md

# excel.runvbcode

## Syntax

```G1ANT
excel.runvbcode code ⟦text⟧
```

## Description

> This command runs a Visual Basic macro code in the currently active Excel instance. The code can be only for Sub procedures, not functions.

Reviewer: "The code can be only for Sub procedures" => "The code can contain only Sub procedures"

Author: sure, but what about people who don't really know what a VBA's Sub is?

Author: ```suggestion
This command runs a Visual Basic macro code in the currently active Excel instance. The code can contain only procedures (Sub blocks), not functions.
```
